libnopegl/src/node_drawother.c

```diff
@@ -722,8 +722,8 @@ static int drawcolor_init(struct ngl_node *node)
         return ret;
 
     const struct ngpu_pgcraft_uniform uniforms[] = {
-        {.name="color",             .type=NGPU_TYPE_VEC3,  .stage=NGPU_PROGRAM_SHADER_FRAG, .data=ngli_node_get_data_ptr(o->color_node, o->color)},
-        {.name="opacity",           .type=NGPU_TYPE_F32,   .stage=NGPU_PROGRAM_SHADER_FRAG, .data=ngli_node_get_data_ptr(o->opacity_node, &o->opacity)},
+        {.name="color",             .type=NGPU_TYPE_VEC3,  .stage= NGPU_PROGRAM_STAGE_FRAG, .data=ngli_node_get_data_ptr(o->color_node, o->color)},
+        {.name="opacity",           .type=NGPU_TYPE_F32,   .stage= NGPU_PROGRAM_STAGE_FRAG, .data=ngli_node_get_data_ptr(o->opacity_node, &o->opacity)},
```

Comment:
whitespace issue

There are actually many other instances of that. I was able to find them all by running the following in a branch:

```sh
git ls-files | xargs sed -i 's/NGPU_PROGRAM_SHADER_/NGPU_PROGRAM_STAGE_/g'
git ls-files | xargs sed -i 's/ngpu_program_shader/ngpu_program_stage/g'
```

...and comparing with your commit.

Reply:
Sorry, did not notice my editor was also trying to apply to some code style at the same time. and since the codebase is a bit inconsistent -> :boom: